content/docs/git/ide.md

```diff
@@ -0,0 +1,248 @@
+
+### 0. Command Line cheat sheet
+
+{{< tip >}}
+Remember that you can look up what a command does by using the `man` (manual-pages) command on Mac and Linux, or `help` command on Windows cmd. For example: `man echo` would give you information on what the `echo` command does and how to use it.
+{{< /tip >}}
+
+| command | what it does                                                                                                                               |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `echo`  | print some text                                                                                                                            |
+| `pwd`   | print working directory (print the path to where I currently am in the file system)                                                        |
+| `ls`    | list the contents of the directory                                                                                                         |
+| `cd`    | change directory (this moves you, the viewer of the terminal, to a new location in the file system)                                        |
+| `cat`   | print the contents of the file                                                                                                             |
+| `cp`    | copy a file (note: to copy a directory you probably need to add an option (like `-r`) to copy a directory)                                 |
+| `mv`    | move a file or directory to a new location (note: this can also be useful for changing the name of a file/directory)                       |
+| `mkdir` | make a directory                                                                                                                           |
+| `touch` | create a new file                                                                                                                          |
+| `rm`    | delete a file (note: to delete a directory, add the option `-r` or `-d`)                                                                   |
+| `>`     | the redirection operator (`X > Y` will send the outputs of command `X` to `Y`; if `Y` exists, it this will _overwrite_ the existing file.) |
+
+
+### 0. Git cheat sheet
+
+These are some of the most famous commands you need.
+
+| command | what it does                                                                                                                               |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `git clone <repo>`  | Clone repo located at <repo> onto local machine. Original repo can be located on the local filesystem or on a remote machine via HTTP or SSH.                                                                                                           |
```

Comment:
maybe we can add a note/example of how to replace the `<...>` syntax -- some students were getting confused and not replacing the `<directory>` with a directory, or were leaving the angle brackets in, up to you though!

Reply:
Aha! I can see that happening! I added a note above 🎉